### DISCUSSION TITLE
Typo on "français" instead of "Français"

### DISCUSSION THREAD

**Discussion by Apyow** [requarks/wiki#6749] [Error / Bug Report] [answered]
Hello,

The native name for French language would be nicer with an uppercase.

![image](https://github.com/requarks/wiki/assets/44231851/7d409bfb-440f-40e9-8c88-1cbc1d8ad540)

I also notice that many native names doesn't have an uppercase for the first character.

This is a detail but beauty is in the details :)

rgds

**Comment by fnafox** [comment]
Not every language uses a capital when referencing the name of the language.

**Reply by Apyow** [reply]
Are there global rules defined somewhere to display languages in an application ?
I think here it's more a question of aesthetics than a standard to apply.

**Reply by fnafox** [reply]
The selector displays the "nativename" string from https://github.com/requarks/wiki-localization/blob/master/locales.json
Native language name strings are displayed in accordance with the grammar of the language in question.

**Reply by Apyow** [reply]
Hum ok I had a look and it seems it's a quite stupid french grammar rule, and I'm french lol.

**Comment by NGPixel** [comment, marked as answer]
The native locale names are provided by Lokalise. Also speaking French, I find the lowercase français to be annoying as well. The provider is being changed in v3 and will display it correctly as "Français".

**Reply by Apyow** [reply]
Cool merci :)
